cmd/frontend/backend/orgs.go

```diff
@@ -12,10 +16,20 @@ import (
 
 var ErrNotAuthenticated = errors.New("not authenticated")
 
+// CheckOrgAccessOrSiteAdminCloud returns an error if:
+// (1) if we are on cloud instance and the user is not a member of the organization
+// (2) if we are on prem instance and
+//    (a) the user is not a member of the organization
+//    (b) the user is not a site admin
+func CheckOrgAccessOrSiteAdminCloud(ctx context.Context, db dbutil.DB, orgID int32) error {
```

Comment:
```suggestion
func CheckOrgAccessOrSiteAdminCloud(ctx context.Context, db database.DB, orgID int32) error {
```

Why don't we ask for this type in the first place?

Reply:
Weird. I'm pretty sure I copy pasted the signature and then modified it a little. Maybe GoLand is playing tricks on me 🙄